sympy/printing/preview.py

```diff
@@ -183,8 +186,9 @@ def preview(expr, output='png', viewer=None, euler=True, packages=(),
     try:
         workdir = tempfile.mkdtemp()
 
-        with open(join(workdir, 'texput.tex'), 'w', encoding='utf-8') as fh:
-            fh.write(latex_main % latex_string)
+        with io.open(join(workdir, 'texput.tex'), 'w', encoding='utf-8') as fh:
+            rendered = latex_main % latex_string
+            fh.write(u(rendered.replace(r'\u', r'\\u')))
```

Comment:
`u()` parses \uXXXX as unicode. hence this line guards LaTeX commands such as `\usepackage`